backend/src/Core/FurnitureShop.Core.Contracts/Mobile/Reviews/ReviewDTOs.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace FurnitureShop.Core.Contracts.Mobile.Reviews
+{
+    public class ReviewDTO
+    {
+        public Guid Id { get; set; }
+        public ReviewInfoDTO ReviewInfo { get; set; }
+    }
+    public class ReviewInfoDTO
+    {
+
+        public Guid? UserId { get; set; }
+        public Guid? ProductId { get; set; }
+        public string? Text { get; set; }
```

Comment:
Be better to have empty text instead of nullable text, it's the same, but database looks better without nulls